plugins/ltac/tactic_debug.ml

```diff
@@ -15,14 +15,16 @@ open Tacexpr
 
 let (ltac_trace_info : ltac_trace Exninfo.t) = Exninfo.make ()
 
+let get_current_context () = let env = Global.env () in Evd.from_env env, env
```

Comment:
wrong env, and misleading function name

Reply:
This should now be removed, shouldn't it?